be/src/common/config.h

```diff
@@ -490,6 +490,9 @@ CONF_mInt32(max_consumer_num_per_group, "3");
 // this should be larger than FE config 'max_concurrent_task_num_per_be' (default 5).
 CONF_Int32(routine_load_thread_pool_size, "10");
 
+// kafka reqeust timeout
+CONF_Int32(routine_load_kafka_timeout_second, "10");
```

Comment:
The default value of is larger than FE config `routine_load_task_consume_second`, which is conflicted.

Reply:
The default value of `routine_load_task_consume_second` is 3.